widget/charts/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@rango-dev/charts",
+  "version": "0.1.0",
+  "license": "MIT",
+  "type": "module",
+  "source": "./src/index.ts",
+  "main": "./dist/index.js",
+  "exports": {
+    "types": "./dist/index.d.ts",
+    "default": "./dist/index.js"
+  },
+  "typings": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "dev": "yarn bundle --watch",
+    "build": "node ../../scripts/build/command.mjs --path widget/charts",
+    "ts-check": "tsc --declaration --emitDeclarationOnly -p ./tsconfig.json",
+    "type-checking": "tsc --declaration --emitDeclarationOnly",
```

Comment:
what is this for?

Reply:
If type-checking is what you mean, Since it comes with copy and paste, we don't need it.